Uno.Gallery/Uno.Gallery.UITest/Given_Chip_01_toolkit.cs

```diff
@@ -0,0 +1,504 @@
+﻿visusing System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Uno.UITest.Helpers.Queries;
+using Uno.UITests.Helpers;
+
+namespace Uno.Gallery.UITests
+{
+	public class Given_Chip_01_toolkit : TestBase
+	{
+
+		/*This function is to test the Header Text and Input Filled - Enabled condition for chip control. */
+		[Test]
+		public void WhenClick_01_InputFilledEnabled()
+		{
+			NavigateToSample("Chip", "Material");
+			//App.Repl();
+			// To verifying the Header Text is correct as "Input -Filled"						
+			var textBlock = new QueryEx(x => x.All().Marked("InputFilledLabel"));
+			Assert.AreEqual("Input - Filled", textBlock.GetDependencyPropertyValue("Text"));
+
+			// Initial verification
+			TakeScreenshot("Before Click");
+			var chipFilledEnabled = new QueryEx(x => x.All().Marked("ChipFilledInput_Enabled"));
```

Comment:
given that we are only interacting with a single chip control, it is fine to just call it `chip`
dont need to be that precise, it is not copy-paste friendly, since you did have to rename it in every test